pipelines/crawler/rf_cnpj/utils.py

```
@@ -0,0 +1,1030 @@
+            f"A data máxima extraida da API da Receita Federal que será utilizada para comparar com os metadados da BD: {folder_date}"
+        )
+    else:
+        try:
+            index = folder_dates.index(
+                next(item for item in folder_dates if folder_date in str(item))
+            )
+            last_modified_date = datetime.datetime.strptime(
+                last_modified_dates[index].find("d:getlastmodified").text,
+                "%a, %d %b %Y %H:%M:%S GMT",
+            ).date()
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)